documentation/source/Introduction/pages/installation.rst

```diff
@@ -189,7 +189,7 @@ These packages are recommended to be able to use all of NuRadioMC/NuRadioReco's
 
   .. code-block:: bash
 
-    pip install proposal==6.1.6
+    pip install proposal
```

Comment:
should we add a `==7.*` here?

Reply:
Good point. Although I hope that we would notify you in advance if there is a major update of PROPOSAL planned :D